bootstrap

```diff
@@ -19,19 +21,22 @@ fi
 
 . "$config_file"
 
-echo "Installing curl and git"
-apt-get update
-apt-get install -y curl git
+if command -v apt-get; then
+	echo "Installing curl and git"
+	apt-get update
+	apt-get install -y curl git
+fi
 
 # Create a tempfile so the user has the opportunity to review the omnitruck script.
-omnitruck_script="$(mktemp --suffix=.bash)"
+omnitruck_script="$(mktemp)"
```

Comment:
```suggestion
omnitruck_script="$(mktemp XXXXXXXXXX.bash)"
```
and you can probably remove the line below. If you have interest into keeping the bash file into the current directory:
```bash
omnitruck_script="$(mktemp -p . XXXXXXXXXX.bash)"
```
but I think that for cleanness we can keep it the temp directory.